subjects/ispowerof2/README.md

```diff
@@ -2,14 +2,18 @@
 
 ### Instructions
 
-Write a program that determines if a given number is a power of 2.
+Write a program that determines if a given number is a power of 2. A number `n` is a power of 2 if it meets the following condition: $n = 2 ^ m$ where m is another integer number.
+
+For example, $4 = 2 ^ 2$ or $16 = 2 ^ 4$ are power of 2 numbers while 24 is not.
```

Comment:
What would you think about rephrasing the example as **4 = 2<sup>2</sup>** and do likewise for what comes after since using the sup tag appears to be supported when doing the preview in the platform?

Reply:
Good catch! I have changed it like you suggested